src/ros_filter.cpp

```diff
@@ -46,6 +48,26 @@
 
 namespace RobotLocalization
 {
+
+  //! @brief Compute covariance volume
```

Comment:
I didn't find a better place to put this utility function. I'd be happy to move it somewhere else into a header file so it can be used somewhere else if needed.